CodeYourFuture/cyf-forms
# Volunteer-team-preference

This is a:






### Description



- **Purpose** - 
- The purpose of this ticket is to enable CYF Staff members to streamline the onboarding process by requesting volunteers to identify the specific team they would like to volunteer for during the sign-up process, this will accelerating the overall onboarding procedure.



- **How to check** - 

- Verify that the list is displaying all the teams {Education, Mentorship, Outreach, Personal Development, Tech Project} in alphabetical order.

- Create a new team in the MongoDB under the "teams" section. You can accomplish this by cloning one of the existing teams and changing its name to "New Team."

- Check the form to ensure that the newly created team ("New Team") is added as an option.

- Navigate to the dashboard at /volunteers and confirm that the "New Team" is correctly added to the teams list.

- Select one user from the dashboard and modify their team assignment, changing it to the newly created "New Team."

- Delete the "New Team" you have created from the MongoDB.

- After deleting the "New Team," verify that the dashboard for the selected user automatically updates to display "Select Team" as the option.

### Links



### Author checklist



- [x] I have made the PR to `qa` from a branch named `<category>/<name>`, e.g. `feature/edit-spaceships` or `bugfix/restore-oxygen`
- [ ] I have completed the manual tests [described here](https://github.com/CodeYourFuture/tech-team/wiki/3.-How-we-work#manual-test-procedures)
